ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -274,6 +270,23 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
     isStackedBar,
   ]);
```

Comment:
Future-looking note (i.e. not asking to fix in this PR). At some point it might be nice to refactor all the tooltip pinning behavior into a hook (1) so it's a little more reusable if we end up having more charts that use tooltips (2) to simplify both this component and the pinning behavior to make it a little easier to maintain over time.

Reply:
This came up in the initial BarChart [PR here](https://github.com/perses/perses/pull/1279#discussion_r1255190068). I definitely want to spend time cleaning up the implementation, but I'm not sure that tooltip pinning is as relevant for other charts--maybe if we add drill-down / ability to filter based on a specific series from inside the tooltip, but in cases where the nearby-series logic and custom highlighting functionality aren't as useful, I think it's fine to use the ECharts tooltip.